src/main/resources/liquibase/tenant/scripts/2025-10-29--11-00-create-configuration-settings.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+
+  <changeSet id="2025-10-29--11-00-create-configuration-settings" author="kjain@ebsco.com">
+
+    <!-- Create configuration_settings table -->
+    <createTable schemaName="${database.defaultSchemaName}" tableName="configuration_settings">
+      <column name="id" type="UUID">
+        <constraints primaryKey="true" primaryKeyName="pk_configuration_settings" nullable="false"/>
+      </column>
+
+      <column name="config_name" type="VARCHAR(100)">
+        <constraints nullable="false" unique="true"/>
+      </column>
+
+      <column name="config_value" type="JSONB">
+        <constraints nullable="false"/>
+      </column>
+    </createTable>
+
+
+    <!-- Insert initial configuration values -->
```

Comment:
Hi Kriti, I'm not sure it is a correct way to insert initial values directly in sql script bypassing /resources/config/*.json files. I think you need to read initial data from the files as it is implemented currently.